api/lib/application/users/index.js

```diff
@@ -76,6 +76,46 @@ exports.register = async function(server) {
         tags: ['api', 'user admin'],
       }
     },
+
+    {
+      method: 'PATCH',
+      path: '/api/admin/users/{id}',
+      config: {
+        pre: [{
+          method: securityPreHandlers.checkUserHasRolePixMaster,
+          assign: 'hasRolePixMaster'
+        }],
+        plugins: {
+          'hapi-swagger': {
+            payloadType: 'form'
+          }
+        },
+        validate: {
+          params: Joi.object({
+            id: Joi.number().integer().required(),
+          }),
+          payload: Joi.any().required().description('User Detail For administration is required'),
+          options: {
+            allowUnknown: true
+          },
+          failAction: (request, h) => {
+            const errorHttpStatusCode = 400;
+            const jsonApiError = new JSONAPIError({
+              code: errorHttpStatusCode.toString(),
+              title: 'Bad request',
+              detail: 'L\'identifiant de l\'utilisateur n\'est pas au bon format.',
+            });
+            return h.response(jsonApiError).code(errorHttpStatusCode).takeover();
+          }
+        },
+        handler: userController.updateUserDetailForAdministration,
+        notes : [
+          '- Met à jour les attributs d\'un utilisateur identifié par son id dans un contexte d\'administration\n',
```

Comment:
Est-ce que ça ne serait pas plus claire ? 
```suggestion
          '- Permet à un administrateur de mettre à jour les attributs d\'un utilisateur identifié par son id',
```

Reply:
Done